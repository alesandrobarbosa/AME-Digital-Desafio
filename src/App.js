import React from 'react';

import Game from './components/Game/game';
import Start from './components/Start/start';
import './view/styles/styles.scss';

import { Switch, Route } from 'react-router-dom';

const App = () => {
  return (
    <div className="main">
      <Switch>
        <Route path="/" exact component={Start}></Route>
        <Route path="/game" component={Game}></Route>
      </Switch> 
      <div className="main__floor"></div>
    </div>
  );

}

export default App;