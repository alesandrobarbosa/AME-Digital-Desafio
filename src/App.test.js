import React from 'react';
import renderer from 'react-test-renderer';

import { MemoryRouter } from 'react-router-dom';
import App from './app';


describe('#APP COMPONENT ==>', () => {

    it('App should renders without crashing', () => {
        const component = renderer.create(
            <MemoryRouter>
                <App />
            </MemoryRouter>
        ).toJSON();
    
        expect(component).toMatchSnapshot();
    });
})
