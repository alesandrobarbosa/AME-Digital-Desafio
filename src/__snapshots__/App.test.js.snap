// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#APP COMPONENT ==> App should renders without crashing 1`] = `
<div
  className="main"
>
  <div
    className="start"
  >
    <img
      alt="logo"
      className="start__logo"
      src="star-wars-logo.png"
    />
    <a
      className="start__play"
      href="/game"
      onClick={[Function]}
    >
      START
    </a>
  </div>
</div>
`;
